src/LowMachEquationSystem.C

```diff
@@ -1106,8 +1177,72 @@ MomentumEquationSystem::register_nodal_fields(
       TAMSAlgDriver_->register_nodal_fields(part);
   }
 
+  if (
+    realm_.solutionOptions_->turbulenceModel_ == SST_IDDES ||
+    realm_.solutionOptions_->turbulenceModel_ == SST_IDDES_ABL) {
+
+    des_lturb_ = &(meta_data.declare_field<ScalarFieldType>(
```

Comment:
@gantech do you have any heart burn with me removing these fields?  It looks like they are only used for the Strelets upwinding computation and are a bunch of intermediate values.

Reply:
Thanks @psakievich Please go ahead and remove them. As long as we have a final working version of IDDES model... I'm more than happy and really glad for the help.